ftests/nuxeo-platform-explorer-ftests/src/test/java/org/nuxeo/functionaltests/explorer/ITExplorerTest.java

```diff
@@ -79,15 +79,14 @@ public void testLoginLogout() {
      */
     @Test
     public void testDistribAdminPage() {
-        open(DistribAdminPage.URL);
-        assertEquals("", driver.getTitle());
-        assertEquals("", driver.getPageSource());
+        open(DistribAdminPage.URL, true);
```

Comment:
I think it would be better to keep the `open()` method as is and have another `openAndCheck404` (or other name) for clarity when reading. The fact that the `true` means "check 404" is not very clear.